xml/containers-get.xml

```diff
@@ -525,51 +497,45 @@
    </listitem>
    <listitem>
     <para>
-     pip packages
+     PyPI packages
     </para>
    </listitem>
    <listitem>
     <para>
-     npm packages
+     NPM packages
     </para>
    </listitem>
   </itemizedlist>
 
   <para>
-   These analyses can be performed on a single image, or a
-   <literal>diff</literal> can be performed on two images to compare.
+   You can inspect a single image, or perform a diff operation on two images.
    <literal>container-diff</literal> supports Docker images located in both a
-   local Docker daemon and a remote registry. Tarballs with the correct
-   extension (.tar, .tar.gz, .tgz) can also be provided to the tool directly.
+   local Docker daemon and a remote registry. It is also possible to use the tool with the <filename>.tar</filename>, <filename>.tar.gz</filename>, <filename>.tgz</filename> archives.
   </para>
 
   <para>
-   The <literal>container-diff</literal> package is part of the {sls} 15 SP2
-   <emphasis role="strong">Containers Module</emphasis> or it can be installed
+   The <package>container-diff</package> package is part of the {sls} 15 SP2
+   <emphasis role="strong">Containers Module</emphasis>. Alternatively, it can be installed
    separately. For instructions on how to install it, see the
    <link xlink:href="https://github.com/GoogleContainerTools/container-diff#installation">container-diff
    documentation</link>.
   </para>
  </sect1>
  <sect1 xml:id="id-on-premise-registry">
-  <title>On-premise Registry</title>
+  <title>On-Premise Registry</title>
```

Comment:
```suggestion
  <title>On-Premises Registry</title>
```

Reply:
The singular is "premises". This is a common mistake but it's still a mistake.

Premises: a place of business.
Premise: a logical proposition upon which an argument is constructed.